pages/instructorQuestionManualGrading/instructorQuestionManualGradingNextInstanceQuestion.js

```diff
@@ -3,45 +3,38 @@ const express = require('express');
 const router = express.Router();
 const path = require('path');
 const debug = require('debug')('prairielearn:' + path.basename(__filename, '.js'));
-const logger = require('../../lib/logger');
 const sqlDb = require('../../prairielib/lib/sql-db');
 const sqlLoader = require('../../prairielib/lib/sql-loader');
 const sql = sqlLoader.loadSqlEquiv(__filename);
 
 router.get('/', (req, res, next) => {
   const params = {
-    assessment_question_id: res.locals.assessment_question_id,
     assessment_id: res.locals.assessment.id,
-    prior_instance_question_id: res.locals.prior_instance_question_id,
+    assessment_question_id: res.locals.assessment_question_id,
+    user_id: res.locals.authz_data.user.user_id,
+    prior_instance_question_id: req.query.prior_instance_question,
   };
 
-  sqlDb.queryZeroOrOneRow(sql.get_next_ungraded_instance_question, params, function (err, result) {
-    if (ERR(err, next)) return;
+  sqlDb.queryZeroOrOneRow(
+    sql.select_next_ungraded_instance_question,
+    params,
+    function (err, result) {
+      if (ERR(err, next)) return;
 
-    // If we have no more submissions, then redirect back to manual grading page
-    if (!result.rows[0]) {
-      logger.info(
-        'ManualGradingNextInstanceQuestion: No more submissions, back to manual grading page.',
-        params
-      );
+      // If we have no more submissions, then redirect back to manual grading page
+      if (!result.rows[0]) {
+        res.redirect(
+          `${res.locals.urlPrefix}/assessment/${res.locals.assessment.id}/assessment_question/${res.locals.assessment_question_id}/manual_grading?done`
```

Comment:
Is the `done` query param used?

Reply:
Leftover from the previous version. I don't think it did anything before, I'm not opposed to removing it.